### PR TITLE
Added new VCN_OpenLoop device class

### PR DESCRIPTION
**Pull request by jozamudi**
## Description

Added VCN_OpenLoop device class to pcdsdevices valve.py. VCN_OpenLoop uses VCN as a template w/ the removal of 'open' and 'position_readback' commands. The 'state' member vairable has been renamed to 'control_mode' and the associated doc string was been updated.

## Motivation and Context

https://jira.slac.stanford.edu/browse/ECSENG-848

## How Has This Been Tested?

Used Typhos to check if device would load correctly.
## Where Has This Been Documented?

Screenshot from 2024-09-24 13-44-06

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: pcdsdevices/valve.py [hunk 1]
```diff
@@ -858,3 +858,40 @@ class VCN_VAT590(BaseInterface, Device):
     )
 
     status = Cpt(VCN_VAT590_Status, '',)
+
+
+class VFV(Device):
+    """
+    VFV = Variable Frequency Valves
+    """
```
Comment (ZLLentz) [inline pcdsdevices/valve.py:866]: Do you know precisely which struct this corresponds to in the vacuum lib? I think it's helpful to include this sort of info here for understanding/clarity purposes (and for review)

Like how the docstring for VCN says:
```
It corresponds to ST_VCN in the lcls-twincat-vacuum library.
```

**Comment by ZLLentz** [comment]
Other than my docs note I see no problems here- I'd like to let Jing's approval gate the PR though

File: docs/source/upcoming_release_notes/848-create_new_VFV_device_class.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- N/A
+
+Device Features
+---------------
+- N/A
+
+New Devices
+-----------
+- VFV: similar to VCN w/ the removal of 'open' and 'position_readback'
+  commands. The 'state' member vairable has been renamed to 'control_mode' and
```
Comment (tangkong) [inline docs/source/upcoming_release_notes/848-create_new_VFV_device_class.rst:19]:
```suggestion
  commands. The 'state' member variable has been renamed to 'control_mode' and
```

**Review by tangkong — COMMENTED** [review]
I agree with zach here, this looks fine to me as long as the PVs are correct and the users are in agreement with it.  I'll let Jing make the final call

File: pcdsdevices/valve.py [hunk 3]
```diff
@@ -858,3 +858,42 @@ class VCN_VAT590(BaseInterface, Device):
     )
 
     status = Cpt(VCN_VAT590_Status, '',)
+
+
+class VFV(Device):
+    """
+    VFV = Variable Frequency Valves
+
+    It corresponds to ST_VCN in the lcls-twincat-vacuum library.
```
Comment (ZLLentz) [inline pcdsdevices/valve.py:867]: Is this true? It uses the same struct as the VCN class and we need a different pcdsdevices class for it? That seems wild to me.

Comment (jozamudi) [reply pcdsdevices/valve.py:868]: Jing suggested we renamed the class to VCN_OpenLoop. The device class will use the ST_VCN with open loop controls.

**Review by jyin999 — APPROVED** [review]
Reviewed.

**Pull request merged**